test/img/opencv/generate.py

```diff
@@ -91,3 +91,72 @@ def writeImg(name, img):
 kernelY = np.float32([[0.4, 0.5, 0.6, -0.3, -0.4]])
 dst = cv.sepFilter2D(img, ddepth=-1, kernelX=kernelX, kernelY=kernelY, borderType=cv.BORDER_REFLECT)
 writeImg('testConvolution.png', dst)
+
+
+# Image perspective warp
+
```

Comment:
Have you executed the script to regenerate the images?

Reply:
Yes, I executed it the way it was described in the README